### ISSUE TITLE
More Info regarding AWS cloud cost integration using Terraform

### ISSUE THREAD

**Issue by prasoon-pxc** [opencost/opencost-website#192]
I want to integrate AWS-cloud cost to opencost but could not able to find out enough documentation about how we can create different stuff like:
- IAM-roles 
- Policies
- Athena Query
- S3-bucket

with terraform

**Comment by vishnu-anil** [comment]
Same here. Doc talks about some Cloudformation template to create the dependent resources but cannot find it anywhere.

**Comment by vishnu-anil** [comment]
> The OpenCost Helm chart doesn't do anything related to IAM, Athena or S3. Is the documentation here insufficient? https://www.opencost.io/docs/configuration/aws

yes its insufficient. What table/columns etc required for athena is not specified anywhere

**Comment by vishnu-anil** [comment]
Anyone looking for this, I was able to create required table and columns using below steps
```
CREATE EXTERNAL TABLE IF NOT EXISTS opencost(
  year VARCHAR(4), month VARCHAR(2), date VARCHAR(2), line_item_usage_start_date DATE, line_item_line_item_type STRING, line_item_resource_id STRING, bill_payer_account_id STRING, line_item_usage_account_id STRING, line_item_product_code STRING, line_item_usage_type STRING, line_item_unblended_cost DOUBLE)
LOCATION 's3://yourbucket/'
```
opencost is the table name I refer inside the json secret in kubernetes
